bcipy/signal/evaluate/evaluate.py

```diff
@@ -0,0 +1,49 @@
+
+"""evaluate.py """
+
+
+from bcipy.signal.evaluate.rules import *
+from bcipy.helpers.load import load_json_parameters
+
+class Evaluator:
+
+	"""Evaluator. Takes in chunks of raw data and tests them against given
+	rules, which raise flags if broken. Feeds warnings to artifact rejector
+	as suggestions. 
+	
+	Add rules given in parameters to evaluator's ruleset and set keys for broken_rules.
+	One heading per rule.
+
+	rules (list of rule objects, defined in rules.py)
+	broken_rules (dict): {'rule name1': True/False (indicating violation), 'rule_name2': T/F, ... , 'rule_namen': T/F}"""
+
+	def __init__(self):
```

Comment:
get necessary parameters for evaluator operation. parameters_dict, voltage_enabled, etc

Reply:
now designed to get params from caller!